solr/core/src/java/org/apache/solr/cli/CLIUtils.java

```diff
@@ -249,12 +247,7 @@ public static String getZkHost(CommandLine cli) throws Exception {
 
     try (SolrClient solrClient = getSolrClient(cli)) {
       // hit Solr to get system info
-      NamedList<Object> systemInfo =
-          solrClient.request(
-              new GenericSolrRequest(SolrRequest.METHOD.GET, CommonParams.SYSTEM_INFO_PATH));
-
-      // convert raw JSON into user-friendly output
-      Map<String, Object> status = StatusTool.reportStatus(systemInfo, solrClient);
+      Map<String, Object> status = StatusTool.reportStatus(solrClient);
```

Comment:
maybe for future, but is it odd a CLIUtils would call StatusTool for a method `reportStatus`?  Should it be moved to CLIUtils instead and the StatusTool should call CLIUtils.reportStatus?

Reply:
makes sense, thanks for digging into this.